### DISCUSSION TITLE
Enable opening and closing folds

### DISCUSSION THREAD

**Discussion by cs-mshah** [alangrainger/share-note#30] [Vote for upcoming features]
Currently although the shared note shows folds next to list items, clicking them doesn't collapse the sub-list items. It would be great to have the shared note share all functionality of the reading view of obsidian. Can we export that as it is?

**Comment by alangrainger** [comment]
Sounds great - I'll get that done this weekend 👍 

>  Can we export that as it is?

Unfortunately, functionality like that is Javascript which is baked into the Obsidian app. So to replicate it I need to reverse-engineer and re-build the required Javascript.

I can't do that for everything all at once, but I can do it for each individual thing, like this.

**Comment by alangrainger** [comment]
All done: 

https://share.note.sx/o5c9py19

Your existing notes will immediately work, you don't need to re-share.

**Comment by andrew-manger** [comment]
@alangrainger this works on the note you shared but has since stopped working (example: https://note.mangerstudios.com/pfw4jfeh#XPwg08X3Z6gW2vxoAkoc/kSReOpcUnipIqgxA1g1OhI)

**Reply by alangrainger** [reply]
This is a different issue - folding headings rather than folding lists. This is now added in **server** version 1.4.0, so you will need to update your self-hosted version to that.
